internal/tfengine/tfengine.go

```diff
@@ -101,3 +102,41 @@ func dump(conf *Config, root, outputPath string) error {
 	}
 	return nil
 }
+
+// backendRE is a regex to capture GCS backend blocks in configs.
+// The 's' and 'U' flags allow capturing multi line backend blocks in a lazy manner.
+var backendRE = regexp.MustCompile(`(?sU)backend "gcs" {.*}`)
+
+// ConvertToLocalBackend converts all the Terraform backend blocks to "local".
+// path should be a path to the root of the configs generated by the Engine.
+func ConvertToLocalBackend(path string) error {
+	// Overwrite root terragrunt file with empty file so it doesn't try to setup remote backend.
+	if err := ioutil.WriteFile(filepath.Join(path, "terragrunt.hcl"), nil, 0664); err != nil {
+		return fmt.Errorf("ioutil.Write terragrunt root file: %v", err)
+	}
+
+	// Replace all GCS backend blocks with local.
+	fn := func(path string, info os.FileInfo, err error) error {
+		if filepath.Ext(path) != ".tf" {
+			return nil
+		}
+
+		b, err := ioutil.ReadFile(path)
+		if err != nil {
+			return fmt.Errorf("ioutil.ReadFile(%q) = %v", path, err)
```

Comment:
Given this is no longer in the test, these logs should not read as a description e.g. read file %q: %v, path, err

Reply:
I changed it, I think in the way that you meant.